### PR TITLE
Example pull request

### PR DESCRIPTION
**Pull request by luizfelippesr**
This is an example pull request

It adds a new file

### CODE REVIEW

**Review by lchamandy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -117,6 +117,8 @@ Bp_gal_answer = run.get_galaxy('Bp', 42)
 in this case `Bp_gal_answer` will be a (number_of_grid_points)x(number_of_redshifts) array.
 
 
+SOMETHING CHANGED HERE
```
Comment (luizfelippesr) [inline README.md:120]: Why did you add this thing here?!
